src/server/helpers/search.js

```diff
@@ -63,6 +63,46 @@ function _searchForEntities(orm, dslQuery) {
 		.then((results) => _fetchEntityModelsForESResults(orm, results));
 }
 
+async function _bulkIndexEntities(entities) {
+	if (entities.length === 0) {
+		return;
+	}
+
+	const bulkOperations = entities.reduce((accumulator, entity) => {
```

Comment:
Isn't this also equivalent to `map` (reduce and push)?

Reply:
Since I'm pushing two elements per loop, I'm not sure I can use map.